libs/shell/controls/src/lib/controls-shell.module.ts

```diff
@@ -9,15 +9,17 @@ import { MatSelectModule } from "@angular/material/select";
 import { RouterModule } from "@angular/router";
 import { GdkModule } from "@gorilainvest/gdk";
 
-
-
-
 import { SlideToggleModule } from '@gorilainvest/ui-toolkit/slide-toggle';
 
```

Comment:
those empty lines seems strange... There a reason to keep it?

Reply:
No, but it is being added automatically by the some pre-hook (prettier I think) so I left it there